### PR TITLE
42389 uikit rebase

### PR DESCRIPTION
**Pull request by rikki-iki**
home-agov

### CODE REVIEW

File: templates/html.html.twig [hunk 1]
```diff
@@ -11,11 +11,19 @@
     <title>{{ head_title|safe_join(' | ') }}</title>
     <css-placeholder token="{{ placeholder_token|raw }}">
     <js-placeholder token="{{ placeholder_token|raw }}">
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
```
Comment (benjy) [inline templates/html.html.twig]: I wonder if we should just add this to agov?

Comment (rikki-iki) [inline templates/html.html.twig]: Not all themes would use it?

**Comment by benjy** [comment]
LGTM

File: templates/html.html.twig (hunk 1 above)
Comment (rikki-iki) [inline templates/html.html.twig]: I've made it a library instead so it's easy to remove

**Pull request merged**